launcher/minecraft/mod/LocalModParseTask.cpp

```diff
@@ -262,6 +263,43 @@ std::shared_ptr<ModDetails> ReadFabricModInfo(QByteArray contents)
     return details;
 }
 
+// https://github.com/QuiltMC/rfcs/blob/master/specification/0002-quilt.mod.json.md#the-schema_version-field
```

Comment:
Would be wise to point to the specific commit that you are implementing the spec from

Reply:
If the intent is updating the spec version with each new revision, then I think it's fine leaving this link as it is, and add a link to the specific commit above the `if (schemaVersion == 1)` line.